.github/workflows/docker-deploy.yaml

```diff
@@ -0,0 +1,31 @@
+name: docker-deploy
+
+on: 
+  push:
+    branches: 
+    - master
+
+jobs:
+  deploy:
+    name: Build and Push Docker
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout JETSCAPE
+      uses: actions/checkout@v2
+      with:
+        path: JETSCAPE-DOCKER
+        
+    - name: Copy Dockerfile
+      run: cp JETSCAPE-DOCKER/docker/Dockerfile-github Dockerfile
+      
+    #https://github.com/marketplace/actions/docker-publish
+    #https://www.docker.com/blog/first-docker-github-action-is-here/
+    - name: Dockerize
+      uses: docker/build-push-action@v1
+      with:
+       username: ${{ secrets.DOCKER_USERNAME }}
+       password: ${{ secrets.DOCKER_PASSWORD }}
+       repository: bardelch/jetscape-public-deploy
```

Comment:
This repository line will need to be changed to target the jetscape dockerhub

Reply:
You can go ahead and make this change already

Reply:
Follow up note on this: another secret is required to specify the target repository in docker.

Reply:
Why does it need to be a secret? It is just the repository name, `jetscape/deploy`, no?

Reply:
Basically, we already have a hammer to solve this problem and this looks like another nail.  

Secrecy is not required but in this case the code can remain constant and we are using the variable mechanism that is provided using repository specific secret information.  There might be a more elegant mechanism that does not require secrecy but may increase complexity in configuration.  I am open to suggestions on what might be a better fit.

Reply:
I see, ok -- wasn't sure if I was missing something or not. Either way seems reasonable to me -- I just added the secret, so we can go ahead with that. Thanks.